src/components/Footer/styles.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable import/no-anonymous-default-export */
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
+import * as m from '../../styles/_mixins'
+import { $lgDesktopBreakpoint } from '../../styles/_variables'
+
+const Footer = styled.footer`
+  background: rgb(240, 8, 235);
+  background: -moz-linear-gradient(
+    163deg,
+    rgba(240, 8, 235, 1) 8%,
+    rgba(192, 26, 242, 1) 20%,
+    rgba(29, 83, 253, 1) 55%,
+    rgba(11, 22, 102, 1) 79%
+  );
+  background: -webkit-linear-gradient(
+    163deg,
+    rgba(240, 8, 235, 1) 8%,
+    rgba(192, 26, 242, 1) 20%,
+    rgba(29, 83, 253, 1) 55%,
+    rgba(11, 22, 102, 1) 79%
+  );
+  background: linear-gradient(163deg, rgba(240, 8, 235, 1) 0%, rgba(29, 83, 253, 1) 35%, rgba(11, 22, 102, 1) 79%);
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr="#f008eb",endColorstr="#0b1666",GradientType=1);
+  min-height: 22rem;
+
+  display: flex;
+  flex-wrap: wrap;
+  align-content: flex-end;
+  padding-bottom: 64px;
```

Comment:
you might need to add padding-right and padding-left here... and make the logo+ cpyright be at end. Now, on my computer look like this.
![image](https://github.com/iPlayed-games/frontend/assets/64808456/84b26075-1f3e-412f-9198-aa6f31796598)

Reply:
but add TODO or BUG comments for later. this is really minor points.

Reply:
TODO comment created.